src/core/trulens/core/utils/requirements.optional.txt

```diff
@@ -0,0 +1,90 @@
+# DEPRECATED; to remove when trulens_eval is removed.
```

Comment:
sorry I don't get it, where are these being used? I know you said in the other PR this was required but where is it required?

Reply:
This is used to give installation instructions if someone tries to use an optional package which is not installed. See `imports.py:format_import_errors`.

Reply:
ah i see